src/Components/Categories.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <section className="categoriesSection">
-    <button type="button">Check Status</button>
-  </section>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <section className="categoriesSection">
+      <button type="button" onClick={() => { dispatch(checkStatus); }}>Check Status</button>
+    </section>
+  );
+};
```

Comment:
- [ ] Kindly fix the dispatch of the `checkStatus` action as now when I press the `Check Status` button nothing is dispatched and the redux state remain the same.